examples/content_management_examples/server/examples/todos/todos.ts

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import {
+  ContentStorage,
+  StorageContext,
+  ContentManagementServerSetup,
+} from '@kbn/content-management-plugin/server';
+import { v4 } from 'uuid';
+import {
+  createInSchema,
+  searchInSchema,
+  Todo,
+  TODO_CONTENT_ID,
+  updateInSchema,
+  TodoSearchOut,
+  TodoCreateOut,
+  TodoUpdateOut,
+  TodoDeleteOut,
+  TodoGetOut,
+  createOutSchema,
+  getOutSchema,
+  updateOutSchema,
+  searchOutSchema,
+} from '../../../common/examples/todos';
+
+export const registerTodoContentType = ({
+  contentManagement,
+}: {
+  contentManagement: ContentManagementServerSetup;
+}) => {
+  contentManagement.register({
+    id: TODO_CONTENT_ID,
+    schemas: {
+      content: {
+        create: {
+          in: {
+            data: createInSchema,
+          },
+          out: {
+            result: createOutSchema,
+          },
+        },
+        update: {
+          in: {
+            data: updateInSchema,
+          },
+          out: {
+            result: updateOutSchema,
+          },
+        },
+        search: {
+          in: {
+            query: searchInSchema,
+          },
+          out: {
+            result: searchOutSchema,
+          },
+        },
+        get: {
+          // TODO: why no get in ?
```

Comment:
Seems like `get` and `delete can strictly accept `id` ? Do I need schema here? Why do we support the schema if not?

Reply:
https://github.com/elastic/kibana/pull/151885#discussion_r1115738350